Userland/Services/WebContent/WebDriverConnection.cpp

```diff
@@ -2534,19 +2524,50 @@ ErrorOr<WebDriverConnection::ScriptArguments, Web::WebDriver::Error> WebDriverCo
 void WebDriverConnection::delete_cookies(Optional<StringView> const& name)
 {
     // For each cookie among all associated cookies of the current browsing context’s active document, un the substeps of the first matching condition:
-    auto* document = m_page_client->page().top_level_browsing_context().active_document();
+    auto* document = current_browsing_context().active_document();
 
-    for (auto& cookie : m_page_client->page_did_request_all_cookies(document->url())) {
+    for (auto& cookie : current_browsing_context().page().client().page_did_request_all_cookies(document->url())) {
         // -> name is undefined
         // -> name is equal to cookie name
         if (!name.has_value() || name.value() == cookie.name) {
             // Set the cookie expiry time to a Unix timestamp in the past.
             cookie.expiry_time = UnixDateTime::earliest();
-            m_page_client->page_did_update_cookie(move(cookie));
+            current_browsing_context().page().client().page_did_update_cookie(move(cookie));
         }
         // -> Otherwise
         //    Do nothing.
     }
 }
 
+// https://w3c.github.io/webdriver/#dfn-calculate-the-absolute-position
+Gfx::IntPoint WebDriverConnection::calculate_absolute_position_of_element(JS::NonnullGCPtr<Web::Geometry::DOMRect> rect)
+{
+    // 1. Let rect be the value returned by calling getBoundingClientRect().
```

Comment:
Can you call `element.get_bounding_client_rect()` here and use the result in steps 3 and 4 below?